lightly/api/api_workflow_datasources.py

```diff
@@ -201,6 +130,79 @@ def download_raw_predictions(
             [('.lightly/predictions/object-detection/image-1.json', 'https://......'),
              ('.lightly/predictions/object-detection/image-2.json', 'https://......')]
 
+        :meta private:  # Skip docstring generation
```

Comment:
Optional: There are two copies of the same docstring. These docstrings are not public, so I would suggest just to make a reference:
```py
 def download_raw_predictions(...):
    """Downloads all prediction filenames and read urls from the datasource.

    See `download_raw_predictions_iter` for details.

    :meta private:  # Skip docstring generation
    """
```